setup.py

```diff
@@ -12,16 +12,16 @@
     py_modules=['tap_exacttarget'],
     install_requires=[
         'funcy==1.9.1',
-        'singer-python==5.9.0',
+        'singer-python==5.12.1',
         'python-dateutil==2.6.0',
         'voluptuous==0.10.5',
         'Salesforce-FuelSDK==1.3.0'
     ],
     extras_require={
         'dev': [
             'ipdb==0.11',
-            'pylint==2.1.1',
-            'astroid==2.1.0',
+            'pylint==2.10.2',
+            'astroid==2.7.3',
```

Comment:
Can you actually separate these into a two groups: `dev` and `test`. The `test` group should have astroid and nose and we should install that group as part of the circleci build.

Reply:
Updated the code.